app/models/marc_fields/bound_with_note_for_access_panel.rb

```diff
@@ -0,0 +1,26 @@
+###
+#  MarcBoundWith class to return 590 fields that have a $c
```

Comment:
Same class comment as for `BoundWithNote`. Might be nice to have a diff comment here for easier grokking of how the classes differ.